share/spack/gitlab/cloud_pipelines/stacks/ml-linux-power64le-cpu/spack.yaml

```diff
@@ -0,0 +1,142 @@
+spack:
+    #  config:
+    #    install_tree:
+    #      projections:
+    #        all: 'morepadding/{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+    #  concretizer:
+    #    unify: false
+  view: false
+  packages:
+    all:
+      require: target=ppc64le
+      variants: ~cuda~rocm
+#    pkgconfig:
+#      require: pkgconf
+#    pil:
+#      require: py-pillow
+#    mpi:
+#      require: openmpi
+#    python:
+#      require: "+libxml2"
+#    py-pybind11:
+#      require: "build_system=cmake"
+#    py-keras-applications:
+#      require: "build_system=python_pip"
+#    curl:
+#      require: "tls=gnutls"
+#    py-pytorch-gradual-warmup-lr:
+#      require: "build_system=python_pip"
+#    py-pytorch-lightning:
+#      require: "build_system=python_pip"
+#    jpeg:
+#      require: "libjpeg-turbo"
+#    py-torch-cluster:
+#      require: "~cuda"
+#    py-torch-geometric:
+#      require: "~cuda"
+#    py-torch-scatter:
+#      require: "~cuda"
+#    py-torch-spline-conv:
+#      require: "~cuda"
+#    py-torchaudio:
+#      require: "build_system=python_pip"
+#    py-torchfile:
+#      require: "build_system=python_pip"
+#    py-torchvision:
+#      require: "build_system=python_pip"
+#    py-numpy:
+#      require: "build_system=python_pip"
+#    py-tensorboard-data-server:
+#      require: "build_system=python_pip"
+#    py-tensorboard-plugin-wit:
+#      require: "build_system=python_pip"
+#    swig:
+#      require: "@4.1.1"
+#    xgboost:
+#      require: "build_system=cmake generator=ninja"
```

Comment:
If these aren't relevant to your stack, let's get rid of them.

Reply:
These were used for local efforts at getting the stacks to concretize.  They will be removed before the PR is ready for final review, but as it is currently marked as `Draft`, I'll leave them until I'm sure they're not required.